aten/src/ATen/native/native_functions.yaml

```diff
@@ -241,7 +235,7 @@
   supports_named_tensor: True
   dispatch:
     CPU: _conj_out_cpu
-    CUDA: _abs_out_cuda
```

Comment:
Hmm, why is this `abs`?

Reply:
perharps @dylanbespalko can answer this question.

Reply:
Hi there,

 I'm working on GPU complex number support for unary ops right now.

The `abs_cuda()` operator is being used as a placeholder for the following ops:
- real()
- imag()
- conj()
- angle()

**I imagine we could just remove the CUDA: dispatch**. There is no complex support for GPU yet. Sorry  for the confusion.

Reply:
Thanks, for porting abs to ATen. I'm bookmarking the PR so that I know how to do this in the future.

Reply:
@dylanbespalko, thanks for your reply.

Reply:
Could you remove these CUDA dispatch according to @dylanbespalko 's suggestion?

Reply:
Removed.